themes/tiddlywiki/vanilla/base.tid

```diff
@@ -627,6 +627,10 @@ button.tc-untagged-label {
 
 .tc-page-controls {
 	margin-top: 14px;
+}
+
+.tc-page-controls > span > button,
+.tc-page-controls > span > span > button{
```

Comment:
Just made some tests. It seems that you can safely remove the whole CSS rule, because it isn't used anywhere. There is no button element in the language switcher dropdown. Have a look at the DOM.